atlp-rwanda/dynamites-ecomm-fe
# landing page banner section

### What does this PR do?
- This PR implements UI the banner section of the landing page using React, Redux, and Tailwind CSS. The banner section will consist of two advertisements images of  products  in database as we wish to to add to ads.

### **Key Changes:**

*   **Components:**
    *   `BAnnerAds Card`: Displays individual product information ( name , short discription,image).
*   **Redux Integration:**
    *   image is fetched using redux  from database 
  
*   **Styling:**
    *   TailwindCSS classes applied for visual consistency with the provided design.
    *   Responsive design ensures a seamless experience across devices.

*   **Testing:**
    *   Thorough component testing to verify rendering and data flow.

### **Screenshots:**
 
**Tests Screenshot**
<img width="604" alt="image" src="https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/110623461/37a05304-b65d-4679-9a54-764a2d0c28f6">
**UI on Mobile**
<img width="239" alt="image" src="https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/110623461/144b0a92-9408-4c8c-8626-c7395d849fd0">
**UI on Tablet
<img width="301" alt="image" src="https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/110623461/2fe1273e-b114-418e-904d-df179566c517">

**UI on Desktop**
<img width="941" alt="image" src="https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/110623461/03ac9620-2148-4978-b630-785b3964b6ba">




### **How to Review:**

1.  Checkout this branch and run the application.
2.  Verify that the landing page matches the provided design.
3.  Test product display on different screen sizes (responsiveness).
4.  Check that product data is fetched correctly from the Redux store.
5.  Review code for clarity, best practices, and adherence to project guidelines.

### Description of the task to be completed?
- [x] Implementthe banner section of the landing page
- [x] Write appropriate tests
- [x] Components integrated with Redux.
- [x] Responsive UI matching the design  for both desktop and mobile screens.
